### DISCUSSION TITLE
how to add custom word dictionary

### DISCUSSION THREAD

**Discussion by shiqin-zhang** [meilisearch/product#331] [Feedback & Feature Proposal]
since meilisearch have already support hanzi (Chinese characters), is there a way to add custom word dictionary for jieba word segmentation ?

**Comment by gmourier** [comment]
Hi @shiqin-zhang! 👋

Today, it is not possible natively, however the library that we use in our tokenizer has a [method to be instantiated from a custom dictionary](https://docs.rs/jieba-rs/latest/jieba_rs/struct.Jieba.html#method.with_dict). This could be a solution to explore!
